shared-lib/spiutils/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+
```

Comment:
Please remove the blank line after `[package]`.

Reply:
Done.